QUARKUS-3290.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3290 - Generate source zip which contains the source code of the built artifacts
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-3290
+
+As part of discussions abound QUARKUS-2701, it was decided to generate source zip which contains the source code of the artifact from the internal Gerrit repository for upcoming 3.x release
```

Comment:
"The artifact" means our main zip for binaries?

Reply:
Should be `artifact => Maven artifacts`

Reply:
Will update the PR ...